index.js

```diff
@@ -18,6 +18,19 @@ function loadSpreadsheet(options, cb) {
     //https://github.com/theoephraim/node-google-spreadsheet
     var my_sheet = new GoogleSpreadsheet(options.key);
 
+    // takes service account info
+    if (options.private_key_id) {
+        options.type = 'service_account';
+        my_sheet.useServiceAccountAuth(options, function () { processSpreadsheet(my_sheet, options, cb) });
```

Comment:
it's better to pass required fields only to `useServiceAccountAuth`
As I can see, it should be something like
{
      client_email: 'yourserviceaccountemailhere@google.com',
      private_key: 'your long private key stuff here'
    }